cypress/features/regression/buttonToggleGroup.feature

```diff
@@ -111,4 +110,10 @@ Feature: Button Toggle Group component
     Examples:
       | direction |
       | left      |
-      | right     |
\ No newline at end of file
+      | right     |
+
+  @positive
+  Scenario: Verify the onChange function for a Button Toggle Group component
+    Given clear all actions in Actions Tab
+    When I click on Button Toggle
+    Then onChange action was called in Actions Tab
```

Comment:
In this case you will check only onChange event for the last button in group as you have 3 clicks on every button

Reply:
Amended to click on then check each button in a separate scenario example 👍 Commented out the middle button though until the removal of the initial 'selected' state on first render has been merged.